examples/rgb20.rs

```diff
@@ -38,7 +38,7 @@ fn main() {
 
     let contract = NonInflatableAsset::testnet("TEST", "Test asset", None, Precision::CentiMicro)
         .expect("invalid contract data")
-        .allocate(Method::TapretFirst, beneficiary, 1_000_000_000_00u64.into())
```

Comment:
This was deliberate to follow the given precision. Better to add `clippy` pragma allowing non-standard number formatting